docs/marionette.region.md

```diff
@@ -398,21 +404,25 @@ This example will make a view slide down from the top of the screen instead of j
 appearing in place:
 
 ```js
+var Marionette = require('backbone.marionette');
+
 var ModalRegion = Marionette.Region.extend({
   attachHtml: function(view){
     // Some effect to show the view:
     this.$el.empty().append(view.el);
     this.$el.hide().slideDown('fast');
   }
-})
+});
 
-MyApp.addRegions({
-  mainRegion: '#main-region',
-  modalRegion: {
-    regionClass: ModalRegion,
-    selector: '#modal-region'
+var MyView = Marionette.View.extend({
+  regions: {
+    mainRegion: '#main-region',
+    modalRegion: {
+      regionClass: ModalRegion,
+      el: '#modal-region'
+    }
   }
-})
+});
 ```
 
 ### Attach Existing View
```

Comment:
This is all gone now too.. you shouldn't set currentView on initialization either.  The behavioral change in v3 is that if the view you are showing is already rendered it won't re-render it.. so a showing a rendered view is just attaching it.